packages/machine-extractor/src/toMachineConfig.ts

```diff
@@ -22,7 +22,7 @@ export interface ToMachineConfigParseOptions {
 
 const parseStateNode = (
   astResult: StateNodeReturn,
-  opts?: ToMachineConfigParseOptions,
+  opts: ToMachineConfigParseOptions | undefined,
```

Comment:
The issue was I wasn't passing the options to every function that needed it

Reply:
That's a good trick to ensure that you always pass params around. I'm using this in Changesets in a couple of places too. 10/10, would recommend to a friend